preconditions/README.md

```diff
@@ -0,0 +1,58 @@
+# IOTICS PRECONDITIONS
+
+In order to guarantee the full streaming functionality of IOTICS, we would need:
+- **HTTP2** to be enabled with your environment;
+- web traffic to be allowed on ports **10001** (gRPC) and **11000** (gRPC-Web).
+- We would also need to ensure the domains `.iotics.space` and `did.prd.iotics.com` will not be blocked
+
+To make these checks easy for you we have prepared a script that performs automatically all of the above tests.
+
+## Linux/MacOS
+
+Execute the `iotics_test.sh` in this folder. If all the tests are successful you will see the following return values:
+
+```bash
+https ---> OK
+grpc-web ---> OK
+gRPC ---> OK
```

Comment:
after a chat with lorenzo, to check for a valid http2 message from back one needs grpc-curl (or similar).
for now this check can be excluded 
alternatively it could be put within a conditional block that executes only if `grpc-curl` is available in the current path

```
GRPC_CURL=$(which grpc-curl)

if [ -z "$GRPC_CURL" ]
then
      echo "grpc-curl not installed: skipping test"
else
      echo "grpc-curl installed: running test"
fi
```

Reply:
I like that approach, even if can't run it shows what we are looking for